cypress/e2e/submit.cy.ts

```diff
@@ -0,0 +1,80 @@
+import enUS from '~/i18n/locales/en.json'
```

Comment:
You should change it here as well to import enUS from "../../i18n/locales/en.json". I could make a ticket that has us fix the config file later but this is the quickfix that I found works.

Reply:
Hmm that's strange, because it should follow the paths defined in the /cypress/tsconfig.json.